### ISSUE TITLE
ESLint Config

### ISSUE THREAD

**Issue by DatMayolein** [rauenzi/BetterDiscordApp#45]
Could you please provide your ".eslintrc" file?
I'd like to optimize your code, but without it, it's nearly impossible :/

**Comment by rauenzi** [comment]
Optimize my code? and there isn't one for this repo

**Comment by DatMayolein** [comment]
But you're using eslint Oo
` /* eslint-disable  no-console */` is in ./js/main.js

Yes optimize:
you're mixing i.Ex. single and double quotes, a windows linebreak-style, weired semicolon placements. So there is a lot to optimize ;)

**Comment by rauenzi** [comment]
That's just due to the global eslint config of my editor, not an eslint for the repo. Yes I'm aware there are several mixing's of things. There is a lot of old code that existed before I forked it that I have not touched. It doesn't optimize or help the system to go and change those things. Also a person's preferred linebreak or semicolon placement doesn't make it not optimized or even necessarily wrong.

**Comment by DatMayolein** [comment]
That's true, but however it's easier to read ;)

**Comment by DatMayolein** [comment]
Uhm... ok. It's your project and your decision. But I recommend you start writing clean code. Not for nothing there are a lot of books for this purpose.

**Comment by rauenzi** [comment]
That's rather rude, I have a master's degree in a relevant field I am aware of what I am doing. Perhaps you should check out BDv2 or my plugin library for "clean code".

**Comment by rauenzi** [comment]
Also as a side note this isn't really _my_ project, it is a fork of a project that fixes several issues, it is not meant to be an entire rewrite and cleanup.

**Comment by DatMayolein** [comment]
I know, I saw the "forked from Jiiks/BetterDiscordApp" note ;)

OT:
I do not want to offend you now and please do not take this as a criticism of you. Just because you have a "master" in one area does not mean you're good at it. You can see such examples every day, whether it's economy, politics or other areas.

I looked at your "BetterDiscordAddons" -> "ReplySystem.plugin.js" but there are i.E. missing Tabs, wrong placed curly brackets, ... too

In summary: I just want to help, give tips and not criticize you

**Comment by rauenzi** [comment]
You do not receive a masters of science degree for being bad in a field. That's just not how the world works. You wouldn't pass the thesis defense.

That plugin you mentioned is transpiled from a properly formatted source. I don't want to take the time to format it myself on every build.

Helping is one thing, but you were just being rude.

**Comment by rauenzi** [comment]
Let me know if that is sufficient for you
